modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/processors/OperationProcessor.java

```diff
@@ -40,8 +39,19 @@ public void processOperation(Operation operation) {
         if(processedOperationParameters != null) {
             operation.setParameters(processedOperationParameters);
         }
-        final RequestBody requestBody = operation.getRequestBody();
+        RequestBody requestBody = operation.getRequestBody();
         if(requestBody != null) {
+        	//This part allows paser to put requestBody inline without the resolveFully option set to true
+        	if (requestBody.get$ref() != null && cache != null && cache.getParseOptions() != null && cache.getParseOptions().isResolveRequestBody()) {
+        		requestBodyProcessor.processRequestBody(requestBody);
+        		RefFormat refFormat = computeRefFormat(requestBody.get$ref());
+        		RequestBody resolvedRequestBody = cache.loadRef(requestBody.get$ref(), refFormat, RequestBody.class);
+
+                if (resolvedRequestBody != null) {
+                	requestBody = resolvedRequestBody;
+                	operation.setRequestBody(resolvedRequestBody);
+                }
+        	}
```

Comment:
Would need to use spaces instead of tabs and correct indentation

Reply:
Done!